samples/bluetooth/direction_finding_connectionless_rx/child_image/hci_rpmsg.conf

```diff
@@ -23,3 +23,10 @@ CONFIG_BT_CTLR_DF_ANT_SWITCH_TX=n
 
 # Disable Direction Finding TX mode
 CONFIG_BT_CTLR_DF_ADV_CTE_TX=n
+
+# Disable Direction Finding connected mode
+CONFIG_BT_CTLR_DF_CONN_CTE_RX=n
+CONFIG_BT_CTLR_DF_CONN_CTE_TX=n
+
+# # limit scanner chains to 1 to fit into the RAM
+CONFIG_BT_CTLR_DF_PER_SCAN_CTE_NUM_MAX=1
```

Comment:
Please add a blank line at the end of this file

Reply:
Added